libraries/lib-viewport/Viewport.cpp

```diff
@@ -422,7 +422,8 @@ void Viewport::DoScroll()
 
    auto width = viewInfo.GetTracksUsableWidth();
    const auto zoom = viewInfo.GetZoom();
-   viewInfo.hpos = std::clamp(sbarH / zoom, lowerBound, total - width / zoom);
+   viewInfo.hpos =
```

Comment:
I think I fixed it on my branch too, hence the conflict

Reply:
Rebased.